sdcm/tester.py

```diff
@@ -1110,6 +1110,158 @@ def collect_partitions_info(self, table_name, primary_key_column, save_into_file
+        return list(session.execute(query))
+
+    def create_role(self, session, role_name, password=None, login=False, superuser=False, options_dict=None,
+                    service_level_name=None, service_level_shares=None):
+        # Example: CREATE ROLE bob WITH PASSWORD = 'password_b'AND LOGIN = true AND SUPERUSER = true;
+        # Example: CREATE ROLE carlos WITH OPTIONS = {'custom_option1': 'option1_value', 'custom_option2': 99};
+        role_options = {}
+        for opt in ['password', 'login', 'superuser', 'options_dict']:
+            if locals()[opt]:
+                role_options[opt.replace('_dict', '')] = locals()[opt]
+        role_options_str = ' AND '.join(['{} = {}'.format(opt, val) for opt, val in role_options.iteritems()])
+        if role_options_str:
+            role_options_str = ' WITH {}'.format(role_options_str)
+
+        query = 'CREATE ROLE IF NOT EXISTS {role_name}{role_options_str}'.format(**locals())
+        self.log.debug('Create role query: {}'.format(query))
+        session.execute(query)
+        self.log.debug('Role "{}" has been created'.format(role_name))
+
+        if service_level_name and service_level_shares:
+            self.create_service_level(session=session, sla_name=service_level_name, shares=service_level_shares)
+            self.attach_service_level(session=session, sla_name=service_level_name, role_name=role_name)
+
+    def grant_roles_to(self, session, role_be_granted, grant_to):
+        query = 'GRANT {role_be_granted} to {grant_to}'.format(**locals())
+        self.log.debug('Grant role query: {}'.format(query))
+        session.execute(query)
+        self.log.debug('Role "{role_be_granted}" has been granted to {grant_to}'.format(**locals()))
+
+    def drop_role(self, session, role_name, if_exists=True):
+        query = 'DROP ROLE{if_exists} {role_name}'.format(role_name=role_name, if_exists=' IF EXISTS' if if_exists else '')
+        self.log.debug('Drop role query: {}'.format(query))
+        session.execute(query)
+        self.log.debug('Role "{}" has been dropped'.format(role_name))
+
+    def revoke_role(self, session, role_be_revoked, revokes_from):
+        query = 'REVOKE ROLE {role_be_revoked} FROM {role_revokes_from}'.format(**locals())
+        self.log.debug('Drop role query: {}'.format(query))
+        session.execute(query)
+        self.log.debug('Role "{role_be_revoked}" has been revocked from {role_revokes_from}'.format(**locals()))
+
+    def attach_another_sla_to_role(self, session, role_name, new_sla_name):
+        current_service_level = self.list_attached_service_levels(session=session, role_name=role_name)
+        self.detach_service_level(session=session, role_name=role_name)
+        self.attach_service_level(session=session, sla_name=new_sla_name, role_name=role_name)
+
+    # def create_user(self, session, user_name, password=None, superuser=None, roles_list_to_grant=None):
```

Comment:
Commented code...

Reply:
I don't want to remove it because of it's preparation for the next test

Reply:
removed